# This file is distributed under the same license as the Django package.
# 
# Translators:
# Ivan Khomutov <iskhomutov@gmail.com>, 2017
# Jannis Leidel <jannis@leidel.info>, 2011
# Алексей Борискин <sun.void@gmail.com>, 2012-2015
# Андрей Щуров <shurov9009@gmail.com>, 2016
# Дмитрий Шатера <mr.bobsans@gmail.com>, 2016
msgid ""
msgstr ""
"Project-Id-Version: django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-07-16 23:42+0300\n"
"PO-Revision-Date: 2017-09-30 20:31+0000\n"
"Last-Translator: Ivan Khomutov <iskhomutov@gmail.com>\n"
"Language-Team: Russian (http://www.transifex.com/django/django/language/"
"ru/)\n"
"Language: ru\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=4; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<12 || n%100>14) ? 1 : n%10==0 || (n%10>=5 && n%10<=9) || (n"
"%100>=11 && n%100<=14)? 2 : 3);\n"

#: .\sanor\Lib\site-packages\django\contrib\auth\admin.py:46
msgid "Personal info"
msgstr "Персональная информация"

#: .\sanor\Lib\site-packages\django\contrib\auth\admin.py:47
msgid "Permissions"
msgstr "Права доступа"

#: .\sanor\Lib\site-packages\django\contrib\auth\admin.py:50
msgid "Important dates"
msgstr "Важные даты"

#: .\sanor\Lib\site-packages\django\contrib\auth\admin.py:134
#, python-format
msgid "%(name)s object with primary key %(key)r does not exist."
msgstr "%(name)s с первичным ключом %(key)r не существует."

#: .\sanor\Lib\site-packages\django\contrib\auth\admin.py:144
msgid "Password changed successfully."
msgstr "Пароль успешно изменен."

#: .\sanor\Lib\site-packages\django\contrib\auth\admin.py:164
#, python-format
msgid "Change password: %s"
msgstr "Изменить пароль: %s"

#: .\sanor\Lib\site-packages\django\contrib\auth\apps.py:15
msgid "Authentication and Authorization"
msgstr "Пользователи и группы"

#: .\sanor\Lib\site-packages\django\contrib\auth\base_user.py:48
msgid "password"
msgstr "пароль"

#: .\sanor\Lib\site-packages\django\contrib\auth\base_user.py:49
msgid "last login"
msgstr "последний вход"

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:40
msgid "No password set."
msgstr "Пароль не задан."

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:45
msgid "Invalid password format or unknown hashing algorithm."
msgstr "Неизвестный формат пароля или алгоритм хеширования."

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:80
#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:321
#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:394
msgid "The two password fields didn't match."
msgstr "Два поля с паролями не совпадают."

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:83
#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:136
#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:172
#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:398
msgid "Password"
msgstr "Пароль"

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:89
msgid "Password confirmation"
msgstr "Подтверждение пароля"

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:92
#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:407
msgid "Enter the same password as before, for verification."
msgstr "Для подтверждения введите, пожалуйста, пароль ещё раз."

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:138
msgid ""
"Raw passwords are not stored, so there is no way to see this user's "
"password, but you can change the password using <a href=\"{}\">this form</a>."
msgstr ""
"Пароли не хранятся в открытом виде, поэтому нет возможности увидеть пароль "
"этого пользователя, но вы можете изменить пароль, воспользовавшись<a href="
"\"{}\">этой формой</a>."

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:179
#, python-format
msgid ""
"Please enter a correct %(username)s and password. Note that both fields may "
"be case-sensitive."
msgstr ""
"Пожалуйста, введите правильные %(username)s и пароль. Оба поля могут быть "
"чувствительны к регистру."

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:182
msgid "This account is inactive."
msgstr "Эта учетная запись отключена."

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:242
msgid "Email"
msgstr "Адрес электронной почты"

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:324
msgid "New password"
msgstr "Новый пароль"

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:330
msgid "New password confirmation"
msgstr "Подтверждение нового пароля"

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:366
msgid "Your old password was entered incorrectly. Please enter it again."
msgstr "Ваш старый пароль введен неправильно. Пожалуйста, введите его снова."

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:369
msgid "Old password"
msgstr "Старый пароль"

#: .\sanor\Lib\site-packages\django\contrib\auth\forms.py:404
msgid "Password (again)"
msgstr "Пароль (еще раз)"

#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:260
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:334
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:430
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:490
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:521
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:557
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:593
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:631
msgid "algorithm"
msgstr "алгоритм"

#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:261
msgid "iterations"
msgstr "итерации"

#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:262
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:340
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:432
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:491
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:522
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:632
msgid "salt"
msgstr "соль"

#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:263
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:341
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:492
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:523
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:558
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:594
#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:633
msgid "hash"
msgstr "хэш"

#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:335
msgid "variety"
msgstr "множество"

#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:336
msgid "version"
msgstr "версия"

#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:337
msgid "memory cost"
msgstr "затраты памяти"

#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:338
msgid "time cost"
msgstr "затраты времени"

#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:339
msgid "parallelism"
msgstr "параллелизм"

#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:431
msgid "work factor"
msgstr "рабочий фактор"

#: .\sanor\Lib\site-packages\django\contrib\auth\hashers.py:433
msgid "checksum"
msgstr "контрольная сумма"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:56
#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:112
msgid "name"
msgstr "имя"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:60
msgid "content type"
msgstr "тип содержимого"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:62
msgid "codename"
msgstr "кодовое название"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:67
msgid "permission"
msgstr "право"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:68
#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:115
msgid "permissions"
msgstr "права"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:122
msgid "group"
msgstr "группа"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:123
#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:219
msgid "groups"
msgstr "группы"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:210
msgid "superuser status"
msgstr "статус суперпользователя"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:213
msgid ""
"Designates that this user has all permissions without explicitly assigning "
"them."
msgstr "Указывает, что пользователь имеет все права без явного их назначения."

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:222
msgid ""
"The groups this user belongs to. A user will get all permissions granted to "
"each of their groups."
msgstr ""
"Группы, к которым принадлежит данный пользователь. Пользователь получит все "
"права, указанные в каждой из его/её групп."

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:230
msgid "user permissions"
msgstr "права пользователя"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:232
msgid "Specific permissions for this user."
msgstr "Индивидуальные права данного пользователя."

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:299
msgid "username"
msgstr "имя пользователя"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:302
msgid "Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
msgstr ""
"Обязательное поле. Не более 150 символов. Только буквы, цифры и символы @/./"
"+/-/_."

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:305
msgid "A user with that username already exists."
msgstr "Пользователь с таким именем уже существует."

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:308
msgid "first name"
msgstr "имя"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:309
msgid "last name"
msgstr "фамилия"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:310
msgid "email address"
msgstr "адрес электронной почты"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:312
msgid "staff status"
msgstr "статус персонала"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:314
msgid "Designates whether the user can log into this admin site."
msgstr ""
"Отметьте, если пользователь может входить в административную часть сайта."

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:317
msgid "active"
msgstr "активный"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:320
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr ""
"Отметьте, если пользователь должен считаться активным. Уберите эту отметку "
"вместо удаления учётной записи."

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:324
msgid "date joined"
msgstr "дата регистрации"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:333
msgid "user"
msgstr "пользователь"

#: .\sanor\Lib\site-packages\django\contrib\auth\models.py:334
msgid "users"
msgstr "пользователи"

#: .\sanor\Lib\site-packages\django\contrib\auth\password_validation.py:102
#, python-format
msgid ""
"This password is too short. It must contain at least %(min_length)d "
"character."
msgid_plural ""
"This password is too short. It must contain at least %(min_length)d "
"characters."
msgstr[0] ""
"Введённый пароль слишком короткий. Он должен содержать как минимум "
"%(min_length)d символ."
msgstr[1] ""
"Введённый пароль слишком короткий. Он должен содержать как минимум "
"%(min_length)d символа."
msgstr[2] ""
"Введённый пароль слишком короткий. Он должен содержать как минимум "
"%(min_length)d символов."
msgstr[3] ""
"Введённый пароль слишком короткий. Он должен содержать как минимум "
"%(min_length)d символов."

#: .\sanor\Lib\site-packages\django\contrib\auth\password_validation.py:112
#, python-format
msgid "Your password must contain at least %(min_length)d character."
msgid_plural "Your password must contain at least %(min_length)d characters."
msgstr[0] "Ваш пароль должен содержать как минимум %(min_length)d символ."
msgstr[1] "Ваш пароль должен содержать как минимум %(min_length)d символа."
msgstr[2] "Ваш пароль должен содержать как минимум %(min_length)d символов."
msgstr[3] "Ваш пароль должен содержать как минимум %(min_length)d символов."

#: .\sanor\Lib\site-packages\django\contrib\auth\password_validation.py:151
#, python-format
msgid "The password is too similar to the %(verbose_name)s."
msgstr "Введённый пароль слишком похож на %(verbose_name)s."

#: .\sanor\Lib\site-packages\django\contrib\auth\password_validation.py:157
msgid "Your password can't be too similar to your other personal information."
msgstr ""
"Ваш пароль не должен совпадать с вашим именем или другой персональной "
"информацией или быть слишком похожим на неё."

#: .\sanor\Lib\site-packages\django\contrib\auth\password_validation.py:185
msgid "This password is too common."
msgstr "Введённый пароль слишком широко распространён."

#: .\sanor\Lib\site-packages\django\contrib\auth\password_validation.py:190
msgid "Your password can't be a commonly used password."
msgstr "Ваш пароль не может быть одним из широко распространённых паролей."

#: .\sanor\Lib\site-packages\django\contrib\auth\password_validation.py:200
msgid "This password is entirely numeric."
msgstr "Введённый пароль состоит только из цифр."

#: .\sanor\Lib\site-packages\django\contrib\auth\password_validation.py:205
msgid "Your password can't be entirely numeric."
msgstr "Ваш пароль не может состоять только из цифр."

#: .\sanor\Lib\site-packages\django\contrib\auth\templates\registration\password_reset_subject.txt:2
#, python-format
msgid "Password reset on %(site_name)s"
msgstr "Сброс пароля на %(site_name)s"

#: .\sanor\Lib\site-packages\django\contrib\auth\validators.py:12
msgid ""
"Enter a valid username. This value may contain only English letters, "
"numbers, and @/./+/-/_ characters."
msgstr ""
"Введите правильное имя пользователя. Это значение может содержать только "
"английские буквы, цифры и знаки @/./+/-/_."

#: .\sanor\Lib\site-packages\django\contrib\auth\validators.py:22
msgid ""
"Enter a valid username. This value may contain only letters, numbers, and "
"@/./+/-/_ characters."
msgstr ""
"Введите правильное имя пользователя. Оно может содержать только буквы, цифры "
"и знаки @/./+/-/_."

#: .\sanor\Lib\site-packages\django\contrib\auth\views.py:158
msgid "Logged out"
msgstr "Не авторизован"

#: .\sanor\Lib\site-packages\django\contrib\auth\views.py:215
msgid "Password reset"
msgstr "Восстановление пароля"

#: .\sanor\Lib\site-packages\django\contrib\auth\views.py:243
msgid "Password reset sent"
msgstr "Письмо с инструкциями по восстановлению пароля отправлено"

#: .\sanor\Lib\site-packages\django\contrib\auth\views.py:252
msgid "Enter new password"
msgstr "Введите новый пароль"

#: .\sanor\Lib\site-packages\django\contrib\auth\views.py:312
msgid "Password reset unsuccessful"
msgstr "Ошибка восстановления пароля"

#: .\sanor\Lib\site-packages\django\contrib\auth\views.py:320
msgid "Password reset complete"
msgstr "Восстановление пароля завершено"

#: .\sanor\Lib\site-packages\django\contrib\auth\views.py:332
msgid "Password change"
msgstr "Изменение пароля"

#: .\sanor\Lib\site-packages\django\contrib\auth\views.py:355
msgid "Password change successful"
msgstr "Пароль успешно изменен"
